coralproject/talk
# [CORL-2394]: Update user drawer account history styles

## What does this PR do?

This removes the restriction of `white-space` `nowrap` to no longer suppress line breaks, which was causing the text to overflow out of the table instead of wrapping. This is for https://vmproduct.atlassian.net/browse/CORL-2394.

## What changes to the GraphQL/Database Schema does this PR introduce?

None

## How do I test this PR?

You can go to the Community tab in the admin and click on any user. Check out their account history. See that all information is visible and not cut off the right side of the user drawer. You may want to add some differing data, such as warning messages, longer usernames, acknowledgment of warning messages, etc., to see how this looks represented in the account history table with these changes.
 
## How do we deploy this PR?
